src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -247,8 +247,8 @@ def get_beacon_config(
 	]
 
 	if boot_cl_client_ctxs != None:
-		cmd.append("--boot-nodes="+",".join([ctx.enr for ctx in boot_cl_client_ctxs]))
-		cmd.append("--trusted-peers="+",".join([ctx.peer_id for ctx in boot_cl_client_ctxs]))
+		cmd.append("--boot-nodes="+",".join([ctx.enr for ctx[:20] in boot_cl_client_ctxs]))
```

Comment:
can we take this out in a constant that we share across the nodes? Easier to update & think of that way.

Reply:
I wouldn't worry about this tbh.

Reply:
i worry about it as I don't want magical numbers floating around; next time we change it we have to change 20 everywhere!

Reply:
fixed.